server/chat-plugins/mafia.ts

```diff
@@ -986,14 +984,17 @@ class Mafia extends Rooms.RoomGame {
 			player.destroy();
 			return;
 		}
-		this.dead[player.id] = player;
+		if (toEliminate in this.playerTable) this.dead[toEliminate] = this.playerTable[toEliminate];
```

Comment:
Should dead players be removed from the `playerTable`?

Reply:
They are, further down in the method

Reply:
Oh, okay.